### PR TITLE
build script errors corrected

### LINKED ISSUE #1896: Upload apk script errors
Upload apk script tries to delete apk branch when it's checked out to the same branch. Makes the build interrupt.

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1896 

**Changes**:Upload script error fixed

### CODE REVIEW

File: scripts/upload-apk.sh [hunk 1]
```diff
@@ -46,19 +45,22 @@ if [ "$TRAVIS_BRANCH" == "$DEVELOPMENT_BRANCH" ]; then
 
     mv app-playstore-release.apk pslab-dev-release.apk
     mv app-fdroid-release.apk pslab-dev-release-fdroid.apk
-    # Push generated apk files to apk branch
 fi
-git checkout apk
-git add -A
+
+git checkout --orphan temporary
+git add --all .
```
Comment (CloudyPadmal) [inline scripts/upload-apk.sh]: git add .

File: scripts/upload-apk.sh [hunk 2]
```diff
@@ -46,19 +45,22 @@ if [ "$TRAVIS_BRANCH" == "$DEVELOPMENT_BRANCH" ]; then
 
     mv app-playstore-release.apk pslab-dev-release.apk
     mv app-fdroid-release.apk pslab-dev-release-fdroid.apk
-    # Push generated apk files to apk branch
```
Comment (CloudyPadmal) [inline scripts/upload-apk.sh]: Put the comment for line 51

File: scripts/upload-apk.sh [hunk 3]
```diff
@@ -46,19 +45,22 @@ if [ "$TRAVIS_BRANCH" == "$DEVELOPMENT_BRANCH" ]; then
...
+git add --all .
 git commit -am "Travis build pushed [$TRAVIS_BRANCH]"
 
+# Delete current apk branch
 git branch -D apk
+# Rename current branch to apk
 git branch -m apk
 
-git push origin apk --force --quiet > /dev/null
+git push origin apk --force --quiet> /dev/null
```
Comment (CloudyPadmal) [inline scripts/upload-apk.sh]: Previous and this same. So revert back to space

**Comment by neel1998** [comment]
@CloudyPadmal @adityastic  can we please merge this.

**Pull request merged**